rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -29,569 +29,550 @@
     # pybind call arguments
-    kwargs_pybind = {"input_image0": inputs[0],"is_output": False,
-                    "crop_area_factor": None, "crop_aspect_ratio": None, "crop_pos_x": None, "crop_pos_y": None, "num_of_attempts": 20}
-    random_cropped_image = b.RandomCrop(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    kwargs_pybind = {"input_image0": inputs[0], "is_output": False,
+                     "crop_area_factor": crop_area_factor, "crop_aspect_ratio": crop_aspect_ratio, "crop_pos_x": crop_pox_x, "crop_pos_y": crop_pox_y, "num_of_attempts": num_attempts, "rocal_tensor_output_layout": rocal_tensor_layout, "rocal_tensor_output_datatype": rocal_tensor_output_type}
+    random_cropped_image = b.RandomCrop(
+        Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return (random_cropped_image)
 
-def rotate(*inputs, angle=None, axis=None, bytes_per_sample_hint= 0, fill_value = 0.0, interp_type = 1, keep_size = False,
-            output_dtype = -1, preserve = False, seed = -1, size = None, device = None):
-    """
-    angle (float) – Angle, in degrees, by which the image is rotated. For 2D data, the rotation is counter-clockwise, assuming top-left corner at (0,0) For 3D data, the angle is a positive rotation around given axis
-
-    axis (float or list of float, optional, default = []) – 3D only: axis around which to rotate. The vector does not need to be normalized, but must have non-zero length. Reversing the vector is equivalent to changing the sign of angle.
-
-    bytes_per_sample_hint (int, optional, default = 0) – Output size hint (bytes), per sample. The memory will be preallocated if it uses GPU or page-locked memory
-
-    fill_value (float, optional, default = 0.0) – Value used to fill areas that are outside source image. If not specified, source coordinates are clamped and the border pixel is repeated.
 
-    interp_type (int, optional, default = 1) – Type of interpolation used.
-
-    keep_size (bool, optional, default = False) – If True, original canvas size is kept. If False (default) and size is not set, then the canvas size is adjusted to acommodate the rotated image with least padding possible
-
-    output_dtype (int, optional, default = -1) – Output data type. By default, same as input type
-
-    preserve (bool, optional, default = False) – Do not remove the Op from the graph even if its outputs are unused.
-
-    seed (int, optional, default = -1) – Random seed (If not provided it will be populated based on the global seed of the pipeline)
-
-    size (float or list of float, optional, default = []) – Output size, in pixels/points. Non-integer sizes are rounded to nearest integer. Channel dimension should be excluded (e.g. for RGB images specify (480,640), not (480,640,3).
-
-    """
+def rotate(*inputs, angle=None, dest_width=0, dest_height=0, interpolation_type=types.LINEAR_INTERPOLATION,
+           rocal_tensor_layout=types.NHWC, rocal_tensor_output_type=types.UINT8):
```

Comment:
Why are the doc strings removed.
We need to add the respectove arguments present

Reply:
The old docstrings were present for only some functions and they contained multiple unused params. I removed the unused params. I'm currently adding docstrings for the new function params